### PR TITLE
docs(common):Fix Readme Template ESP #378

### LINKED ISSUE #378: Fix Broken Link
https://guias.eoscostarica.io/docs/herramientas/plantilla-readme/

Haga click aqui is a broken link

image

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?
Fixed broken link and updated readme.

- resolves #378 

### Steps to test
1. Go to https://guias.eoscostarica.io/docs/herramientas/plantilla-readme and click on 'haga click aqui'

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/herramientas/plantilla-readme-codigo-fuente.md [hunk 1]
```diff
@@ -6,155 +6,132 @@ keywords: [código fuente, plantilla, README, Ejemplos de README]
...
 
 ```
-    ## Introducción con GIF o Imágenes
+## Introducción con GIF o Imágenes
 
-    Es simple y puede parecer muy fácil y sí, lo es, pero el impacto que puede producir un GIF o una imagen es demasiado grande. El espectador adquiere la información necesaria para interesarse por el proyecto. Por ejemplo:
+Es simple y puede parecer muy fácil y sí, lo es, pero el impacto que puede producir un GIF o una imagen es demasiado grande. El espectador adquiere la información necesaria para interesarse por el proyecto. Por ejemplo:
 
-    ![Life bank](/img/life-bank.webp)
+![Captura de pantalla de Life bank](/img/life-bank.png)
```
Comment (xavier506) [inline docs/herramientas/plantilla-readme-codigo-fuente.md:13]: webp format is better

File: docs/herramientas/plantilla-readme.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ keywords: [Plantilla Readme, Readme, markdown, EOS, EOS Costa Rica]
 README es un archivo que trae una pequeña descripción sobre todo el proyecto, que contiene una introducción y explicación de este. El tipo de información que se puede describir aquí es información solicitada para comprender el proyecto y su estructura o propósito. A continuación, brindamos una plantilla para crear un archivo README para un nuevo proyecto.
 
 
-### Para Copiar esta Plantilla Readme como Código Fuente Markdown [haga clic aquí](plantilla-readme-codigo-fuente)
+### Para Copiar esta Plantilla Readme como Código Fuente Markdown [haga clic aquí](./plantilla-readme-codigo-fuente.md)
```
Comment (xavier506) [inline docs/herramientas/plantilla-readme.md:12]: 👍

File: docs/herramientas/plantilla-readme.md [hunk 3]
```diff
@@ -9,15 +9,15 @@ keywords: [Plantilla Readme, Readme, markdown, EOS, EOS Costa Rica]
...
 
 ***
 
 ## Introducción con GIF o Imágenes
 
 Es simple y puede parecer muy fácil y sí, lo es, pero el impacto que puede producir un GIF o una imagen es demasiado grande. El espectador adquiere la información necesaria para interesarse por el proyecto. Por ejemplo:
 
-![Captura de pantalla de Life bank](/img/life-bank.webp)
+![Captura de pantalla de Life bank](/img/life-bank.png)
```
Comment (xavier506) [inline docs/herramientas/plantilla-readme.md:20]: webp format is better than png

File: docs/herramientas/plantilla-readme-codigo-fuente.md (hunk 1 above)
Comment (fvives99) [reply docs/herramientas/plantilla-readme-codigo-fuente.md]: changed

File: docs/herramientas/plantilla-readme.md (hunk 3 above)
Comment (fvives99) [reply docs/herramientas/plantilla-readme.md]: changed.

**Review by xavier506 — DISMISSED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**